spec/features/idv/in_person_spec.rb

```diff
@@ -58,7 +58,7 @@
     expect(page).to have_text(InPersonHelper::GOOD_DOB)
     expect(page).to have_text(InPersonHelper::GOOD_STATE_ID_NUMBER)
     expect(page).to have_text(InPersonHelper::GOOD_ADDRESS1)
-    expect(page).to have_text(InPersonHelper::GOOD_ADDRESS2)
+    expect(page).to have_text(InPersonHelper::GOOD_ADDRESS2) if InPersonHelper::GOOD_ADDRESS2
```

Comment:
I expect constants to be ... constant? like we should know if the value exists or not ahead of time? Like we would just remove this line or no?

Reply:
Yeah, there's some indirection because this constant is ultimately derived from some other constants. But I am happy to just yank it; that makes sense too.